<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewGame</class>
 <widget class="QDialog" name="DialogNewGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../res.qrc">:/img/res/new_game_dialog.png</pixmap>
   </property>
  </widget>
  <widget class="QCheckBox" name="timerCheck">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>77</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="assistCheck">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>217</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="timeSpin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="incrementSpin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>255</y>
     <width>149</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>10</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
